util/libcrypto.num

```diff
@@ -4917,3 +4917,6 @@ PKCS8_pkey_add1_attr                    ?	3_0_0	EXIST::FUNCTION:
 PKCS8_pkey_add1_attr_by_OBJ             ?	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_private_check                  ?	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_pairwise_check                 ?	3_0_0	EXIST::FUNCTION:
+DH_generate_ffc_parameters              ?	3_0_0	EXIST::FUNCTION:DH
+DSA_generate_ffc_parameters             ?	3_0_0	EXIST::FUNCTION:DSA
+DSA_generate_parameters_ctx             ?	3_0_0	EXIST::FUNCTION:DSA
```

Comment:
It seems like a bad idea to make these public.

Reply:
Removed...